test/Twig/Tests/Profiler/Dumper/AbstractTest.php

```diff
@@ -13,32 +13,88 @@
 {
     protected function getProfile()
     {
-        $profile = new Twig_Profiler_Profile();
-        $index = new Twig_Profiler_Profile('index.twig', Twig_Profiler_Profile::TEMPLATE);
-        $profile->addProfile($index);
-        $body = new Twig_Profiler_Profile('embedded.twig', Twig_Profiler_Profile::BLOCK, 'body');
-        $body->leave();
-        $index->addProfile($body);
-        $embedded = new Twig_Profiler_Profile('embedded.twig', Twig_Profiler_Profile::TEMPLATE);
-        $included = new Twig_Profiler_Profile('included.twig', Twig_Profiler_Profile::TEMPLATE);
-        $embedded->addProfile($included);
-        $index->addProfile($embedded);
-        $included->leave();
-        $embedded->leave();
-
-        $macro = new Twig_Profiler_Profile('index.twig', Twig_Profiler_Profile::MACRO, 'foo');
-        $macro->leave();
-        $index->addProfile($macro);
-
-        $embedded = clone $embedded;
-        $index->addProfile($embedded);
-        usleep(500);
-        $embedded->leave();
-
-        usleep(4500);
-        $index->leave();
-
-        $profile->leave();
+        $profile = $this->getMockBuilder('Twig_Profiler_Profile')->disableOriginalConstructor()->getMock();
+
+        $profile->method('isRoot')->willReturn(true);
```

Comment:
It seems you are using Prophecy or a newer version of PHPUnit.
Twig still supports PHP 5.2 which means you can't use the last PHPUnit version.
https://travis-ci.org/twigphp/Twig/jobs/56346922